api/views/phones.py

```diff
@@ -890,6 +910,32 @@ def _handle_sms_reply(
     relay_number.save()
 
 
+def _send_sms_reply_iq(
+    relay_number: RelayNumber, destination_number: str, body: str
+) -> None:
+    _send_iq_sms(relay_number.number, destination_number, body)
+    relay_number.remaining_texts -= 1
+    relay_number.texts_forwarded += 1
+    relay_number.save()
```

Comment:
suggestion (blocking): To make it easy to understand how sending reply or forwarding inbound messages updates the `RelayNumber` limits put the logic to increment the `texts_forwarded` and decrement the `remaining_texts` either at `inbound_sms_iq` or at `_send_iq_sms`

Reply:
I plan to re-use `_send_iq_sms` for other messages (e.g., phone confirmation and welcome), so I'm moving these lines back out into the calling `inbound_sms_iq` function.